### ISSUE TITLE
Plugin renders empty page on Ubuntu with 5.2.8.50

### ISSUE THREAD

**Issue by michaeldiederich** [zspitzer/lucee-performance-analyzer#2]
I installed the latest `extension-PerformanceAnalyzer-1.0.0.3.lex` first in the server context, then in my web context. After a lucee restart, I see the Analyser in Menu but there is no content:

![image](https://user-images.githubusercontent.com/15325153/78529898-ecf4c580-77e2-11ea-96d3-1cbf5f329466.png)

Lucee: 5.2.8.50
OS: ubuntu 18

What do I wrong?

**Comment by zspitzer** [comment]
do you see any exceptions if you append &debug=true to the url?

**Comment by michaeldiederich** [comment]
No, nothing.

**Comment by zspitzer** [comment]
What does the url look like for that page?

**Comment by michaeldiederich** [comment]
http://projectname/lucee/admin/server.cfm?action=plugin&plugin=PerformanceAnalyzer&debug=true

I clicked through the project and refreshed the lucee admin page after each request.

I also checked the admin/web.cfm with the same url and the debug parameter.

**Comment by zspitzer** [comment]
can you try appending `&pluginAction=scopes` to the url?

the default action is overview, which should just redirect to the scopes action

https://github.com/zspitzer/lucee-performance-analyzer/blob/master/source/cfml/plugins/overview.cfm#L1

**Comment by michaeldiederich** [comment]
I needed to enable alle Debugging in the Web Server Context (was set to now and the Server Administrator Debug was set to yes) and I need to add the parameter in der URL, so now I see the output for SCOPES and QUERIES. For LOGS i get this error message:

```
key [scope] doesn't exist (existing keys:pageParts,starttime,queries,traces,implicitAccess,pages,datasources,cgi,timers,id,history,dumps,exceptions)

error occured in /home/projectname/WEB-INF/lucee/context/admin/plugin/PerformanceAnalyzer/logs.cfm:73
called by /home/projectname/WEB-INF/lucee/context/admin/plugin/Plugin.cfc:84
called by /home/projectname/WEB-INF/lucee/context/admin/plugin/PerformanceAnalyzer/Action.cfc:38
called by /admin/plugin.cfm:62
called by /admin/web.cfm:430
```

URL is: http://projectname/lucee/admin/web.cfm?action=plugin&plugin=PerformanceAnalyzer&pluginAction=Logs&

Thanks so far!

**Comment by zspitzer** [comment]
that's due to 5.2 having a different debug log structure than 5.3, the latest version should fix it

**Comment by drossi70** [comment]
Hi Zac, I installed the Performance Plug to Lucee version (**5.3.5.92**).
**Performance analyzer: Installed version1.0.0.4 | Installed version | 1.0.0.4**

Enabled the Debug at server level and at Context website level.
I get the website url with parameters &debug=true&pluginAction=scopes
I saw the debug information in the page in the Classic Template view.
When I go to the Lucee Admin / Plugin/ Performance section , I Get "No logs found? Is debugging enabled?"
No errors are reported.

If I can help, tell me what I can try, thanks a lot!!!

**Comment by zspitzer** [comment]
Do you see logs under the built in Debugging - Logs?

**Comment by drossi70** [comment]
Under Debugging Logs I see only: 
Configure the size of the debug logs.
Maximum Logged Requests = 200

**Comment by zspitzer** [comment]
ok, that's the problem, both pages use the same backend api

are you looking at the server or web admin?

maybe try just enabling debug at the web context level

**Comment by drossi70** [comment]
OK, Enabling the debug at the COntext level, now I see the Debug Log correctly to the Context Admin interface.
But, the application Plugin Performance is not visibile in the Context Admin
Also, to the Main Admin, the performance plugin don't see the debug log.

**Comment by zspitzer** [comment]
Yeah, that's super annoying, server context plugins aren't accessible at the web context

I have been working on a fix for this
https://github.com/lucee/Lucee/pull/901

Please vote for this issue
https://luceeserver.atlassian.net/browse/LDEV-2791

**Comment by zspitzer** [comment]
closing as this works in more recent versions

**Comment by michaeldiederich** [comment]
The problem still occours for me :(
Lucee 5.3.7.48
I see the logs in the web admin but the plugin is still empty

**Comment by michaeldiederich** [comment]
Lucee 5.4.3.2

Installed the plugin in the web context, enabled full debuging with template, the plugin-page is empty. If I add the URL parameter &pluginAction=scopes it works.
